.cirrus.yml

```diff
@@ -0,0 +1,10 @@
+version_check_task:
+  name: Test versiones go
+  env:
+    matrix:
+      - GO_VERSION: 1.19
+      - GO_VERSION: 1.18
+  container:
+    image: golang:${GO_VERSION}-alpine
+  get_script: go get ./...
+  build_script: go build ./...
```

Comment:
Luis, me vas a perdonar porque te lo curras y además con buena disposición. Pero aquí hay dos problemas
1. No usas el task manager. La idea del task manager es precisamente esa, que lo uses con órdenes estándar para llevar a cabo las tareas comunes, sin tener que acordarte de la sintaxis específica. Como en todos objetivos, lo que se hace en uno sirve para otro; el objetivo 3 sirve para el 4 (añadir la realización de tests) y para el 5 (usar `loquesea` test en vez de una orden específica del lenguaje o toolchain elegido). También, obviamente, para el 6, que es en que estamos.
2. El problema mayor, sin embargo, es que no estás haciendo test, ni con el task runner ni con nada. Se trata de que hagáis test, eso es explícitamente para lo que sirve el sistema de CI elegido.

Reply:
Entiendo, me pongo con ello :ok_hand:

Reply:
Buenas @JJ, he estado mirando para usar make en Cirrus CI y por lo que estoy viendo, usa un script en las máquinas virtuales de forma que las limitan para solo usar instrucciones especificas de estas ya que he probado distintas formas de usar make y ninguna funciona, por otro lado he añadido la ejecución de los test y a pesar de que funcionan,  al principio también me daba un fallo porque usaba gcc y al igual que make tampoco se puede usar si bien añadiendo una etiqueta  lo deshabilita y no hay problema.
Que hago ahora ¿Busco un nuevo CI o lo dejo como está?

Reply:
¿Ves como hay que listar los requisitos antes de optar por una opción u otra?
Haz lo que veas; pero si Cirrus permite descargar contenedores Docker, podrías mirar las especificaciones de GitHub actions a ver si, no sé, por casualidad, tuviera make ya instalado de serie.

Reply:
Me pongo con ello :ok_hand:

Reply:
Buenas @JJ! He continuado mirando y no era problema de Cirrus, lo que he puesto anteriormente de la seguridad no es verdad  ya que si permite usar más cosas, el problema es que los contenedores de alpine que usaba, no tenían instalado make.
No he encontrado como instalarlo, pero he buscado más contenedores y los de bullseye si que vienen con make y por lo tanto ya funciona :confetti_ball: